### PR TITLE
Fixes #21028 - make k-c-h --help work even without an installed system

### PR DESCRIPTION
**Pull request by evgeni**
/etc/foreman-installer/scenarios.d/last_scenario.yaml might or might not
be there, so let's fall back to `katello` scenario if the file cannot be
found.

### CODE REVIEW

**Comment by evgeni** [comment]
thanks @theforeman-bot, but f7c3f1c is not the latest commit in this PR, it's 984e955

**Comment by evgeni** [comment]
rebased

File: katello/helper.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module KatelloUtilities
   module Helper
     def last_scenario
-      File.basename(File.readlink("/etc/foreman-installer/scenarios.d/last_scenario.yaml")).split(".")[0]
+      File.basename(File.readlink("/etc/foreman-installer/scenarios.d/last_scenario.yaml")).split(".")[0] rescue 'katello'
```
Comment (johnpmitsch) [inline katello/helper.rb]: is there a way we can avoid a rescue? Maybe check if the file exists?

**Review by johnpmitsch — CHANGES_REQUESTED** [review]

File: katello/helper.rb (hunk 1 above)
Comment (evgeni) [reply katello/helper.rb]: Yeah, that might be possible as well.

**Comment by evgeni** [comment]
@johnpmitsch updated!

**Comment by ehelms** [comment]
These packages and code have all moved to foreman-packaging `rpm/develop` under a `katello` subfolder. Please re-open your PR against it to ensure there is no data loss between the repositories. The code in this repository will be subsequently removed -- https://github.com/Katello/katello-packaging/pull/575

**Comment by evgeni** [comment]
moved to https://github.com/theforeman/foreman-packaging/pull/1929

**Pull request closed without merging**